### ISSUE TITLE
Vertical axis: Add `Axis.labels.overflow` support

### ISSUE THREAD

**Issue by lionel87** [highcharts/highcharts#13573]
#### Expected behaviour
Chart area does not clip and does not have spacing either.

#### Actual behaviour
Chart area gets clipped at the top.

#### Live demo with steps to reproduce
https://jsfiddle.net/lionel87/tdfj3vrx/
The 175k at the top is half visible.

The bug is more visible when i set a plot border:
https://jsfiddle.net/lionel87/tdfj3vrx/4/

#### Product version
Highcharts v8.1.0

#### Affected browser(s)
Any.

**Comment by pawelfus** [comment]
Hi @lionel87 

Thank you for reporting the issue. I'm not sure if we should consider this as a bug. Border does not increase the spacing, for example: https://jsfiddle.net/BlackLabel/azusne3q/ - half (5px) of the top border is cut off and the other half, rendered inside the plotting area, is visible. In your case, instead of setting `spacing=0`, I suggest to set `spacing=1` (or `spacingTop=1` and others to 0).

**Comment by lionel87** [comment]
Hi @pawelfus 

Thanks for the fast reply, I get it now!

The last label (175k) is just vertically centered at the last tick. So it's half visible, because the last tick is exactly at the top.

This is not a bug then, I just expected something different what Highcharts does now. Maybe because on xAxis the last label is not centered but right aligned when it would clip.

Like at this example here, the 2017 label on xAxis is kept inside the plot area: https://jsfiddle.net/lionel87/34gfpbrL/

In this case a handy feature would be to align the last tick vertically on yAxis like xAxis does horizontally with its last tick (`xAxis.labels.overflow = "justify"`).
Feel free to close this ticket or make a feature request from it if you see this fit in Highcharts.

**Comment by pawelfus** [comment]
Yes, that's correct, `labels.overflow` could improve that label's position. Note that `overflow` does work on both, xAxis and yAxis, but only when the axis is horizontal. See for example inverted chart: https://jsfiddle.net/BlackLabel/dswqxzp7/

Changing the ticket name then, thanks!

**Comment by pawellysy** [comment]
unstale
